static/js/settings_linkifiers.js

```diff
@@ -65,6 +84,7 @@ export function populate_linkifiers(linkifiers_data) {
     }
 
     const linkifiers_table = $("#admin_linkifiers_table").expectOne();
+    $(linkifiers_table).attr("data-status", "saved");
```

Comment:
What's the story with this `data-status` attribute?  We don't AFAIK use similar attributes anywhere except our standard `settings_save_discard_widget.hbs` widget; are you trying to replace its UI?